ide/utils/mailinglist.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import mailchimp
+
+from django.conf import settings
+
+
+class MailingList(object):
```

Comment:
This feels like a class for the sake of making a class; it has no state and I don't think it would be less clear if `add_user` were just a top-level function of this module.